### DISCUSSION TITLE
Halide with webGL/GPU support

### DISCUSSION THREAD

**Discussion by HarishAgarwal88** [halide/Halide#5568] [General]
Does halide support webGL and webGPU? Or are there any plans?

**Comment by shoaibkamil** [comment]
We don't currently support either of these.  WebGL's model (without compute shaders) makes it difficult to correctly map Halide semantics.  Some of us have discussed supporting WebGPU, and we'd certainly welcome contributions in that direction.

**Reply by HarishAgarwal88** [reply]
Thanks for the clarity.
